### DISCUSSION TITLE
Is there any way to tell HomeKit not to stream audio?

### DISCUSSION THREAD

**Discussion by sokulski** [Sunoo/homebridge-camera-ffmpeg#1096] [Q&A] [answered]
I've noticed that when adding a camera to HomeKit via this plugin, even if I leave "Enable Audio" unchecked, the Home app still behaves as if there is sound coming form the camera. It'll turn down whatever audio is playing on my computer when I view the camera.

Is there any way to tell HomeKit that the camera doesn't have audio so that it doesn't try to stream it?

**Comment by Sunoo** [comment, marked as answer]
I believe all real HomeKit cameras stream audio, so Home just assumes there will be an audio stream.
